cmd/influx/main.go

```diff
@@ -83,56 +135,41 @@ func (c *cli) connect(cmd string) {
 	}
 }
 
-func (c *cli) parseCommand(cmd string) {
-	lcmd := strings.ToLower(cmd)
-	switch {
-	case strings.HasPrefix(lcmd, "exit"):
-		os.Exit(0)
-	case strings.HasPrefix(lcmd, "gopher"):
-		gopher()
-	case strings.HasPrefix(lcmd, "connect"):
-		c.connect(cmd)
-	case strings.HasPrefix(lcmd, "pretty"):
-		c.pretty = !c.pretty
-		if c.pretty {
-			fmt.Println("Pretty print enabled")
-		} else {
-			fmt.Println("Pretty print disabled")
-		}
-	case strings.HasPrefix(lcmd, "use"):
-		fmt.Println("We should use a database specified now... todo")
-	default:
-		c.executeQuery(cmd)
-	}
-}
-
 func (c *cli) executeQuery(query string) {
 	results, err := c.client.Query(client.Query{Command: query, Database: c.database})
 	if err != nil {
 		fmt.Printf("ERR: %s\n", err)
+		return
 	}
-	for _, r := range results.Results {
-		var i interface{}
-		if r.Err != nil {
-			i = r.Err
-		} else {
-			if len(r.Rows) == 0 {
-				continue
-			}
-			i = r.Rows
-		}
-		var data []byte
-		if c.pretty {
-			data, err = json.MarshalIndent(i, "", "    ")
-		} else {
-			data, err = json.Marshal(i)
-		}
-		if err != nil {
-			fmt.Printf("ERR: %s\n", err)
-			return
-		}
-		fmt.Fprintln(os.Stdout, string(data))
+	var data []byte
+	if c.pretty {
```

Comment:
Perhaps you don't need to do this -- you can just pass `pretty=true` on the URL.